.eslintrc.js

```diff
@@ -0,0 +1,137 @@
+module.exports = {
+  env: {
+    es6: true,
+    browser: true,
+  },
+  ignorePatterns: [
+    "**/dist/",
+    "**/htmlcov/**",
+    "**/node_modules/",
+    "*.config.js",
+    "*.json",
+    "*.lock",
+    ".eslintrc.js",
+    ".venv/",
+    "build/",
+    "docs/",
+    "package.json",
+    "package-lock.json",
+  ],
+
+  extends: [
+    "airbnb",
+    "airbnb/hooks",
+    "eslint-config-preact",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: "module",
+    project: "./tsconfig.json",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  plugins: ["@typescript-eslint", "prefer-arrow", "prettier"],
+  rules: {
+    // leave these to prettier
+    "comma-dangle": "off",
+    "implicit-arrow-linebreak": "off",
+    "max-len": "off",
+    "no-confusing-arrow": "off",
+    "no-console": "off",
+    "no-multiple-empty-lines": "off",
+    "object-curly-newline": "off",
+    "operator-linebreak": "off",
+    "quote-props": "off",
+    "react/jsx-closing-bracket-location": "off",
+    "react/jsx-curly-newline": "off",
+    "react/jsx-indent": "off",
+    "react/jsx-indent-props": "off",
+    "react/jsx-one-expression-per-line": "off",
+    "wrap-iife": "off",
+
+    // overly strict rules
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "function-paren-newline": "off",
+    "jsx-a11y/label-has-associated-control": "off",
+    "jsx-a11y/no-noninteractive-element-interactions": "off",
+    "jsx-a11y/no-static-element-interactions": "off",
```

Comment:
I wasn't sure if we'd maybe want re-enable these? They enforce some accessibility standards, e.g. not attaching (user-)event listeners to static elements like `div`s unless we have given that element an `aria-role` of (for example) `button`.
I disabled them in `integreat-cms` because these rules were broken a *lot*.

Reply:
I don't have a strong opinion here, but I guess it would be fine to start with a very strict config and only if you notice you spend a lot of your developing time fixing debatable warnings, we could disable them again...